.gitignore

```diff
@@ -3,4 +3,5 @@
 static/css/fit.css
 *.sqlite
 Pipfile
-Pipfile.lock
\ No newline at end of file
+Pipfile.lock
+.idea/**
```

Comment:
Might be a better idea to add specific editor configurations to your global `.gitignore` file instead of the project repo.